base/exports.jl

```diff
@@ -950,6 +950,7 @@ export
     sizeof,
 
 # tasks and conditions
+    channeled_tasks,
```

Comment:
As a convenience function, Yes and Yes. Makes it easy to link related tasks with channels in a single call. Effectively performs 

- create a set of Channels
- create a set of tasks
- bind tasks to channels
- schedule tasks

in a single call.